### PR TITLE
Add coma-dangle and array-bracket-spacing eslint rules and fix errors

### PR DESCRIPTION
**Pull request by JamyDev**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -10,7 +10,9 @@
         "no-param-reassign": 0,
         "space-before-function-paren": [2, "always"],
         "vars-on-top": 0,
-        "no-underscore-dangle": 0
+        "no-underscore-dangle": 0,
+        "comma-dangle": [2, "always-multiline"],
+        "array-bracket-spacing": [2, "always"]
```
Comment (connor4312) [inline .eslintrc]: This contradicts the style on most of our other projects

Comment (JamyDev) [inline .eslintrc]: Wut, I thought we used that everywhere?

Comment (JamyDev) [inline .eslintrc]: Fixed.

**Comment by JamyDev** [comment]
Wait now I'm confused why are there no changes and what happened to my local copy...

**Comment by JamyDev** [comment]
AirBnb please, they changed the default comma rule >_>

**Comment by UnwrittenFun** [comment]
sure

File: .eslintrc [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
-        "no-underscore-dangle": 0
+        "no-underscore-dangle": 0,
+        "comma-dangle": [2, "always-multiline"],
+        "array-bracket-spacing": [2, "never"]
     },
     "globals": {
         "Uint8Array": true,
         "Uint32Array": true,
         "ArrayBuffer": true,
         "DataView": true,
```
Comment (UnwrittenFun) [inline .eslintrc]: rm comma maybe?

**Comment by SimonSchick** [comment]
lgtm

**Pull request merged**